Pcap++/src/DpdkDevice.cpp

```diff
@@ -8,6 +8,11 @@
 #include "DpdkDevice.h"
 #include "DpdkDeviceList.h"
 #include "Logger.h"
+#include "rte_version.h"
+#if (RTE_VER_YEAR >= 16) && (RTE_VER_MONTH > 07)
+#include "rte_bus_pci.h"

```

Comment:
This file doesn't exist below DPDK 17.11 and is causing DPDK 16.11 to fail. Could you please fix this?